test/e2e/run_device_test.go

```diff
@@ -89,10 +89,7 @@ var _ = Describe("Podman run device", func() {
 	})
 
 	It("podman run device host device with --privileged", func() {
-		if _, err := os.Stat("/dev/kvm"); err != nil {
-			Skip("/dev/kvm not available")
-		}
-		session := podmanTest.Podman([]string{"run", "--privileged", ALPINE, "ls", "/dev/kvm"})
+		session := podmanTest.Podman([]string{"run", "--privileged", ALPINE, "ls", "/dev/kmsg"})
```

Comment:
Could you enhance the test to make sure /dev/kmsg does not exists without the --privileged flag.

Reply:
Yes, but I'd like to do that in a separate PR since getting the new images in place seems much more important...opened #11164